templates/rhpam74-authoring-ha-datagrid.yaml

```diff
@@ -381,7 +381,7 @@ parameters:
 - displayName: KIE Server Startup Strategy
   description: "When set to OpenShiftStartupStrategy, allows KIE server to start up independently used shared state from OpenShift API service. Default: ControllerBasedStartupStrategy"
   name: KIE_SERVER_STARTUP_STRATEGY
-  value: ControllerBasedStartupStrategy
+  value: OpenShiftStartupStrategy
```

Comment:
If we are taking away the ability to configure a controller, and only allowing for OSSS, why make KIE_SERVER_STARTUP_STRATEGY here configurable as a template parameter?  If it's not really an option, then we should hardcode it in the kieserver DeyploymentConfig.

Reply:
Correct me if I'm wrong. For 7.4 release, our goal is to set the default startup strategy to OSSS, rather than completely remove 'Controller'. It should be the case in future release.

Reply:
We have two options:

1. Leave the startup strategy parameter configurable, but then also leave the controller parameters configurable, in the case that the user decides to change KIE_SERVER_STARTUP_STRATEGY from OpenShiftStartupStrategy to ControllerBasedStartupStrategy.
2. Don't make the startup strategy parameter configurable, but then there is no purpose in exposing the controller parameters, as there would be no way to change the startup strategy to make them relevant.

If we do truly want the default to be OSSS, but still allowing the customer to change to CBSS, then the controller parameters should not have been removed. Otherwise there's no point in having the startup strategy configurable.

Reply:
Updated as per option #2. Since the 'SmartRouter' does not compatible with OSSS enabled KIE Server, relate template parameter and env variables are removed from those KIE Server only templates.